test.js

```diff
@@ -0,0 +1,63 @@
+const { readdirSync } = require('fs');
+const { join } = require('path');
+const exec = require('async-execute');
+
+/**
+ * fixtures directory
+ * @type {string}
+ */
+const fixtures = join(__dirname, 'fixtures');
+
+/**
+ * eslint binary
+ * @type {string}
+ */
+const eslint = join(__dirname, 'node_modules/.bin/eslint');
+
+/**
+ * this test case config file
+ * @type {string}
+ */
+const config = join(__dirname, '.eslintrc');
+
+/**
+ * Output of stream output
+ * @type {object}
+ */
+const result = {
+    status: undefined
+};
+
+/**
+ * Populate the variable 'result' with execution result
+ * @param {string} file to check
+ */
+const lint = async(file) => await exec(
+    [
+        eslint,
+        '-c',
+        config,
+        `"${join(__dirname, 'fixtures', file)}"`
+    ].join(' ')
+).then(
+    () => { result.status = 'pass'; }
+).catch(
+    () => { result.status = 'fail'; }
+);
+
+describe('linting tests', () => {
+    afterEach(() => {
+        result.status = undefined;
+    });
+    readdirSync(fixtures).forEach((fixture) => {
```

Comment:
Did you mean `it.each(...)`?

Reply:
Actually I didn't. I wanted it to go:
- first fail
- first pass
- second fail
- second pass

And now it'll go
- first fail
- second fail
- first pass
- second pass

But since you dragged yourself all the way over here to honor us with your presence, I'll use Jest's stupid iterator. Whatever

Reply:
🤣